Kiss.Bff/Intern/Seed/Features/SeedController.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Text.Json;
+using Kiss.Bff.Beheer.Data;
+using Kiss.Bff.Beheer.Gespreksresultaten.Data.Entities;
+using Kiss.Bff.Beheer.Links.Data.Entities;
+using Kiss.Bff.NieuwsEnWerkinstructies.Data.Entities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Kiss.Bff.Intern.Seed.Features
+{
+    [ApiController]
+    [Route("api/seed")]
+    [Authorize(Policy = Policies.RedactiePolicy)]
```

Comment:
Er is een test die checkt op de aanwezigheid van de policy: AuthorizationCheckTests in Kiss.Bff.Test. 
Deze methode graag toevoegen in de functie 'GetControllersMethodsWithDefaultAuthorizeAttributes' 

En dan graag de naam ervan meteen aanpassen naar GetControllersMethodsWithRedactiePolicyAuthorizeAttribute 😁

Reply:
Test aangepast